jekyll/_cci2/contexts.md

```diff
@@ -237,121 +237,110 @@ to set up your CircleCI command line interface._
+containing the plaintext secret as a string.
 
-#### Deleting Context Environment Variables
+#### Deleting Environment Variables
 
 ##### Using CircleCI’s CLI
 {:.no_toc}
 
-To delete an environment variable using our CLI, perform the following steps:
-
-1. If you have not already done so, find the context name that contains the environment variable you wish to delete. Execute this command in the CLI:\
-  `circleci context list <vcs-type> <org-name>`
-2. Confirm the environment variable exists within that context. Execute this command in the CLI to list all variables under that context:\
-  `circleci context show <vcs-type> <org-name> <context-name>`
-3. Delete the environment variable by executing this command:\
-  `circleci context remove-secret <vcs-type> <org-name> <context-name> <secret name>`
-
-##### Using CircleCI’s API
-{:.no_toc}
+To delete an environment variable using the CLI, perform the following steps:
 
-To delete an environment variable using our API, perform the following steps:
+1. If you have not already done so, find the context name that contains the 
+   environment variable you wish to delete. Execute this command in the CLI:
+   `circleci context list <vcs-type> <org-name>`
 
-1. Call the [Delete Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext) endpoint in one of our supported languages. For this request, replace the `context-id` and the `env-var-name` with the ID for the context and the environment variable name that should be updated. 
+2. Confirm the environment variable exists within that context. Execute this 
+   command in the CLI to list all variables under that context:
+   `circleci context show <vcs-type> <org-name> <context-name>`
 
-    To see request samples in all supported languages, visit our [api docs on creating an environment variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext).
+3. Delete the environment variable by executing this command:
+   `circleci context remove-secret <vcs-type> <org-name> <context-name> <env-var-name>`
 
-2.  If the request is successful, the response will include a message object
-    confirming the variable was successfully deleted. An unsuccessful response
-    will include a clear error message explaining the error reported.
+##### Using CircleCI’s API
+{:.no_toc}
 
+To delete an environment variable using the API, call the [Delete Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext) endpoint.
 
-#### Rotating Context Environment Variables
+For this request, replace the `context-id` and the `env-var-name` with the ID 
+   for the context and the environment variable name that should be updated.
 
+#### Rotating Environment Variables
 
-Rotation occurs when an old API token is replaced with a new token. 
+Rotation refers to the process of updating a secret's value without deleting it 
+or changing its name.
```

Comment:
We should be careful to use the term "secret" rather than "token" when referring to our env vars; they often contain tokens, but not always, so that could be confusing to users - esp. bc the more common use of that term in our case is to describe CircleCI API tokens and VCS OAuth tokens.